MediaBrowser.Providers/Lyric/TxtLyricProvider.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Lyrics;
+using MediaBrowser.Controller.Resolvers;
+
+namespace MediaBrowser.Providers.Lyric;
+
+/// <summary>
+/// TXT Lyric Provider.
+/// </summary>
+public class TxtLyricProvider : ILyricProvider
+{
+    /// <inheritdoc />
+    public string Name => "TxtLyricProvider";
+
+    /// <summary>
+    /// Gets the priority.
+    /// </summary>
+    /// <value>The priority.</value>
+    public ResolverPriority Priority => ResolverPriority.Second;
+
+    /// <inheritdoc />
+    public IReadOnlyCollection<string> SupportedMediaTypes { get; } = new[] { "lrc", "elrc", "txt" };
+
+    /// <summary>
+    /// Opens lyric file for the requested item, and processes it for API return.
+    /// </summary>
+    /// <param name="item">The item to to process.</param>
+    /// <returns>If provider can determine lyrics, returns a <see cref="LyricResponse"/>; otherwise, null.</returns>
+    public LyricResponse? GetLyrics(BaseItem item)
+    {
+        string? lyricFilePath = this.GetLyricFilePath(item.Path);
+
+        if (string.IsNullOrEmpty(lyricFilePath))
+        {
+            return null;
+        }
+
+        string[] lyricTextLines = System.IO.File.ReadAllLines(lyricFilePath);
```

Comment:
Why can't you add `using System.IO;` to the file?
Also, would it make sense to make this method async?

Reply:
> Why can't you add using System.IO; to the file?

Done

> Also, would it make sense to make this method async?

I don't know. Is there a good way to determine if it would be helpful?

Reply:
I took a look at making GetLyrics() async. By doing this I'll need to update ILyricProvider, which leads to updating LRCLyricProvider.

I got the following error in LRCLyricProvider.
```
Parameters or locals of type 'ReadOnlySpan<char>' cannot be declared in async methods or async lambda expressions.
```

Do I need to create a new interface for GetLyrics() to allow it to be async and overload ILyricProvider? Unsure it it's worth all the effort.

Reply:
You can get around the limitations by putting the span stuff in a local function